### ISSUE TITLE
How to generate char  mask for V2 of coco Text. 

### ISSUE THREAD

**Issue by crazysal** [andreasveit/coco-text#24]
the website mentions 
Mask Annotations
Segmentation mask is annotated for every word, allowing fine-level detection.

how to generate those ?

**Comment by crazysal** [comment]
Or do you mean only the binary word level bbox masks as in used for localisation iou ?

**Comment by andreasveit** [comment]
Thank you for your interest in coco text. For V2 of the dataset, please refer to 
https://github.com/bgshih/coco-text

**Comment by crazysal** [comment]
Thanks for the quick reply @andreasveit , I did come from the v2 repo. Unfortunately as it is a fork, there doesn't seem to be a way to open issues @bgshih could mayb create it as an independent repo . 
I am currently working on an end-to-end scene text recognition model. 
And for now I conceded with working with Synthtext as my project needed char box level annotation which I assumed was there in coco-text but it only has the word bbox level segmentation masks. Though the need is for a pre-training step and any model pre-trained on SynthText requires unending amounts of fine-tuning for generalization to other datasets like that of ICDAR as the it just ends up learning any background texture as the text pixels... .    
Please advice.
Would it be possible to generate binary masks otherwise as one could for coco panoptic segmentation task using the coco-api. I could try extracting character contour masks from those.

**Comment by ZDDEAN** [comment]
请问您有将synthtext数据集格式转换为icdar格式的脚本？
